core/spec/lib/spree/core/controller_helpers/order_spec.rb

```diff
@@ -2,12 +2,13 @@
 
 require 'rails_helper'
 
-class FakesController < ApplicationController
-  include Spree::Core::ControllerHelpers::Order
-end
-
 RSpec.describe Spree::Core::ControllerHelpers::Order, type: :controller do
-  controller(FakesController) {}
+  controller(ApplicationController) {
+    include Spree::Core::ControllerHelpers::Store
+    include Spree::Core::ControllerHelpers::Pricing
+    include Spree::Core::ControllerHelpers::Auth
+    include Spree::Core::ControllerHelpers::Order
```

Comment:
It's not evident to me why we need to include all these modules now, can you explain the reason?

Reply:
Previously there was the FakesController class that was shared among all controller helpers spec files. During the file loading (but before the specs were ran) each helper was including itself into that class. So, at the time the specs were ready to run, the FakesController class had all the helpers included. 

Now, instead, should be more clear on which helpers each helper depends, e.g. the `Order` helper needs methods from `Store`, `Pricing`, and `Auth`.